README.md

```diff
@@ -75,6 +75,14 @@ For client side (benchmark runners):
     # or make install INSTALL_DIR=~/.local/bin
     ```
 
+1. Install Java workers:
+    ```bash
+    cd javaclient
+    for d in tg-benchmark-client-*; do cd $d && mvn package; cd ..; done
```

Comment:
@mliszcz, these installation lines are quite a bash script. What do you think about adding a javaclient-install.sh to javaclient folder?

Reply:
Fixed in 985e089802d64ae02ec1e167acf4704af96649c6